docs/flatcar-linux/packet.md

```diff
@@ -110,6 +110,10 @@ module "controller" {
   # Check yours from the `IPs & Networks` tab from your Packet.net account.
   # If an IP block is not allocated yet, try provisioning an instance from the console in
   # that region. Packet will allocate a public IP CIDR.
+  # Note: the private CIDR range has been observed to change unexpectedly, 
+  # although at the time it was an empty/dormant Packet project and there had been some networking upgrades
+  # be sure to manually check this value again when either reusing an older project 
+  # or see unexplainable networking behaviour (inter-node timeouts)
```

Comment:
What about something more clear (doormant Packet project is not a term I'm familiar with)? Like this:

```suggestion
Note: Packet does not guarantee this CIDR to be stable if there are no servers deployed in the project and region
```

Reply:
dormant means asleep, unused. But sure, that's also fine/clearer :)